src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -84,6 +96,56 @@ const getVipValidationsById = (
   }, {});
 };
 
+const SubnetHelperText: React.FC<{ matchingSubnet: HostSubnet; cluster: Cluster }> = ({
+  matchingSubnet,
+  cluster,
+}) => {
+  const excludedHosts =
+    cluster?.hosts?.filter(
+      (host) => !matchingSubnet.hostIDs.includes(host.requestedHostname || ''),
+    ) || [];
+
+  if (excludedHosts.length === 0) {
+    return null;
+  }
+
+  return (
+    <Alert
+      title="This subnet range is not available on all hosts"
+      variant={AlertVariant.warning}
+      isInline
+    >
+      Hosts outside of this range will not be included in the new cluster.
+      <br />
+      <Popover
+        position="right"
+        bodyContent={
+          <ul>
+            {excludedHosts
+              .sort(
+                (hostA, hostB) =>
+                  hostA.requestedHostname?.localeCompare(hostB.requestedHostname || '') || 0,
+              )
+              .map((host) => (
+                <li key={host.id}>{host.requestedHostname}</li>
+              ))}
+          </ul>
+        }
+        minWidth="30rem"
+        maxWidth="50rem"
+      >
+        <Button
+          variant={ButtonVariant.link}
+          id="form-input-hostSubnet-field-helper-view-excluded"
+          isInline
+        >
+          {`View ${excludedHosts.length} not matching host${excludedHosts.length > 1 ? 's' : ''}`}
```

Comment:
```suggestion
          {`View ${excludedHosts.length} affected host${excludedHosts.length > 1 ? 's' : ''}`}
```

`affected` further emphasizes that these hosts that will be impacted in some way, which is probably good (and is a little less backend-ey).